frontend/lib/src/theme/utils.ts

```diff
@@ -398,12 +398,15 @@ export const createTheme = (
   )
 
   const emotion = createEmotionTheme(completedThemeInput, startingTheme)
+  const basewebTheme = cloneDeep(
```

Comment:
Can you add a comment here that explains the root issue a bit why we are doing it and add that its more of a temporary solution until we find out whats really going on here.